fdb-record-layer-lucene/src/main/java/org/apache/lucene/queryparser/classic/QueryParser.java

```diff
@@ -0,0 +1,1050 @@
+/* QueryParser.java */
+/* Generated By:JavaCC: Do not edit this line. QueryParser.java */
+package org.apache.lucene.queryparser.classic;
```

Comment:
It looks like we're checking in both `QueryParser.java` and `QueryParser.jj`. Does running javacc generation on `QueryParser.jj` generate `QueryParser.java`? And is the idea that if people edit `QueryParser.jj` that they check in the new `QueryParser.java`?